app/.pipeline/config.js

```diff
@@ -71,12 +73,9 @@ const phases = {
     instance: `${name}-dev-${deployChangeId}`,
     version: `${deployChangeId}-${changeId}`,
     tag: `dev-${version}-${deployChangeId}`,
-    host:
-      (isStaticDeployment && (staticUrls.dev)) ||
-      `${name}-${changeId}-a0ec71-dev.apps.silver.devops.gov.bc.ca`,
+    host: (isStaticDeployment && staticUrls.dev) || `${name}-${changeId}-a0ec71-dev.apps.silver.devops.gov.bc.ca`,
```

Comment:
Would a ternary be more readible here? I'm pretty sure it is functionally the same but I'm not certain

Reply:
Should be the same in the end

That being said, there is definitely more to clean up in all of these pipeline files, especially around all of these variables, and how they get passed around (its a bit hard to trace them through the steps)

But maybe in another ticket